R/getPlot.R

```diff
@@ -0,0 +1,121 @@
+#' Get a plot
+#'
+#' Creates and retrieves a PNG of a plot for an FcsFile. Displays
+#' the image if in RStudio, unless "raw" is specified.
```

Comment:
This should document all possible return types. Something like this:
```suggestion
#' Gets a plot for an FcsFile. If in RStudio or Jupyter and `raw` is not `TRUE`,
#' displays the image. Otherwise, returns the raw, PNG-encoded bytes.
```